suite-native/device-manager/src/components/DeviceAction.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactNode } from 'react';
+
+import { HStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const contentStyle = prepareNativeStyle(utils => ({
+    paddingHorizontal: utils.spacings.medium,
+    paddingVertical: 12,
+    alignItems: 'center',
+    height: 44,
+    gap: utils.spacings.small,
+    backgroundColor: utils.colors.backgroundSurfaceElevation1,
+    borderWidth: utils.borders.widths.small,
+    borderRadius: 12,
+    borderColor: utils.colors.borderElevation1,
+}));
+
+export const DeviceAction = ({ children }: { children: ReactNode }) => {
```

Comment:
Hmm, I wonder if the Pressable could be part of this component as well..

Reply:
🤔 it might. I'll think about how to do that